### ISSUE TITLE
Unable to create player configuration from a json

### ISSUE THREAD

**Issue by xxsnakerxx** [bitmovin/bitmovin-player-ios-sdk-cocoapod#10]
There is no method `fromJSON` in `BMPPlayerConfiguration` class. Do you have any plans to add it?

**Comment by hawk23** [comment]
Yes, we have plans to add this. It is scheduled for one of the upcoming releases, but at the moment we cannot provide a more specific timeline. We'll let you know as soon as this feature is available.
